spec/e2e/has_one_spec.rb

```diff
@@ -24,6 +24,12 @@
       it 'returns a nil object' do
         expect(unsaved_node.parent).to eq nil
       end
+
+      describe 'with chainable: true option' do
+        it 'returns an empty association proxy object' do
+          expect(unsaved_node.parent(chainable: true).inspect).to eq '#<AssociationProxy HasOneB#parent []>'
```

Comment:
I think this would be a bit better as:

```
expect(unsaved_node.parent(chainable: true)).to be_a Neo4j::ActiveNode::HasN::AssociationProxy
```

I can make that change

Reply:
https://github.com/neo4jrb/neo4j/commit/a34cb4cc49cde56a73d5974315e45526356d9a76